content/docs/1_guide/7_languages/1_introduction/guide.txt

```diff
@@ -76,16 +76,22 @@ If you don't use the Panel, you have to add the language extension to your text
 
 ### Via the Panel
 
-After enabling languages in your config as described above, you can find the language setup on the Panel in the "Settings" page.
+In versions prior to 3.6.0, you can find the language setup in the "Settings" page. Managing languages works in the same way.
```

Comment:
Based our discussion of today - creating separate docs copies e.g. for 3.5 - I would suggest that we remove anything about pre-3.6 versions here.